tests/test_templates.py

```diff
@@ -33,8 +35,8 @@ def test_get_TestEnvironment(self):
         with signac.TemporaryProject(name=gen.PROJECT_NAME) as p:
             fp = gen.get_masked_flowproject(p)
             fp.import_from(origin=gen.ARCHIVE_DIR)
+            jobs = fp.find_jobs(dict(environments=self.env_name()))
```

Comment:
```suggestion
            jobs = fp.find_jobs(dict(environment=self.env_name()))
```
I think this is causing the error. Because of wrong argument it can't find any jobs.